google/protobuf/duration.proto

```diff
@@ -0,0 +1,104 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Unfortunately grpc bazel rules don't work with well known protos yet. Added a TODO to get rid of local copy when grpc supports this capability.